react-toggle-switch-component/hannah-app/src/ToggleButton.jsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import './App.css';
+
+export default function ToggleButton({ label, toggled }) {
+  const [isOn, setIsOn] = useState(toggled);
+
+  function handleToggle() {
+    setIsOn(!isOn);
+  }
+
+  return (
+    <>
+      <label>
+        <input type="checkbox" defaultChecked={isOn} onClick={handleToggle} />
+        <span />
+        {label}
```

Comment:
Interesting. It could be useful to put a label here. Did you try it? What does it look like?
The way this PR currently is structured, you are not passing a `label` prop to this component. In App.tsx you are only passing the `toggled` prop.